taichi/ir/statements.h

```diff
@@ -428,6 +428,26 @@ class ExternalFuncCallStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE
 };
 
+/**
+ * Call an external (C++) function.
+ */
+class CallCppStmt : public Stmt {
```

Comment:
Can we re-use `ExternalFuncCallStmt`?

Reply:
https://github.com/taichi-dev/taichi/pull/2873/commits/58ab97f629fd52cebb13fb35a0feb599a0af3fe1